app/main.py

```diff
@@ -1,6 +1,20 @@
+import os
+
+import requests
+from dotenv import load_dotenv
+
+load_dotenv()
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    print("Performing request to Weather API for city paris...")
+    request = requests.get(f"http://api.weatherapi.com/v1/current.json?"
+                           f"key={os.getenv('API_KEY')}&q=Paris&aqi=no")
+
+    temp = request.json()["current"]["temp_c"]
+    weather = request.json()["current"]["condition"]["text"]
+    time = request.json()["current"]["last_updated"]
+    print(f"Paris/France {time} Weather: {temp} Celsius, {weather}")
```

Comment:
Why Use a try Block Here?
Network Errors: The requests.get call can fail due to network issues, timeouts, or invalid URLs. Wrapping it in a try block ensures that these errors are caught and handled gracefully rather than causing your program to crash.

HTTP Errors: If the API returns an error response (e.g., 4xx or 5xx status codes), using response.raise_for_status() will raise an exception. This exception is caught by the except block to provide a meaningful error message.

JSON Parsing Errors: The response.json() call can fail if the response is not valid JSON. Wrapping this in a try block allows you to catch ValueError and handle it appropriately.

Missing Keys: The code assumes that certain keys will always be present in the JSON response. If the API response format changes or if the data is missing, this can raise a KeyError, which you can catch and handle to avoid crashes.

Unforeseen Errors: Catching a generic Exception ensures that any other unexpected errors are also handled, providing robustness to your function.